File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/History.java
package work.socialhub.kmastodon.entity;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * @author uakihir0
 */
public class History implements Serializable {

    @SerializedName("day")
    private String day;
    @SerializedName("uses")
    private Long uses;
    @SerializedName("accounts")
    private Long accounts;

    public String getDay() {
        return day;
    }

    public void setDay(String day) {
        this.day = day;
    }

    public Long getUses() {
        return uses;
    }

    public void setUses(Long uses) {
        this.uses = uses;
    }

    public Long getAccounts() {
        return accounts;
    }

    public void setAccounts(Long accounts) {
        this.accounts = accounts;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/History.kt
package work.socialhub.kmastodon.entity

import kotlinx.serialization.Serializable
import kotlin.js.JsExport

@JsExport
@Serializable
class History {
    var day: String? = null
    var uses: Int? = null
    var accounts: Int? = null
}
